keywords.robot

```diff
@@ -5,6 +5,7 @@ Library         osfv-scripts/osfv_cli/osfv_cli/snipeit_robot.py
 Resource        keys-and-keywords/flashrom.robot
 Resource        pikvm-rest-api/pikvm_comm.robot
 Resource        lib/secure-boot-lib.robot
+Resource        lib/power-after-fail-lib.robot
```

Comment:
Is it needed anywhere else besides `dasharo-compatibility/power-after-fail.robot`? I don't think so. it could be imported directly in `dasharo-compatibility/power-after-fail.robot` only. Note that you create a circular include:

1. `lib/power-after-fail-lib.robot` includes `keywords.robot`
2. `keywords.robot` includes `lib/power-after-fail-lib.robot`

In such case, does `lib/power-after-fail-lib.robot` need to include `keywords.robot`? I thought `keywords.robot` is included in every test suite

Reply:
Fixed in [recent commit](https://github.com/Dasharo/open-source-firmware-validation/commit/e84e768780bae0482929f7309386c111a401257c)